README.md

```diff
@@ -44,8 +44,21 @@ $ brew install dnsmasq
 $ sudo brew services start dnsmasq
 ```
 
+### What services might you have available? Let's find out! ###
+
+```
+$ brew services list
+Name       Status  User Plist
```

Comment:
In an ideal world they would all show output examples (provides point of reference for noobs if something goes wrong), but in this case I agree; I'll update the PR and nuke those output examples.